website/style-dark.css

```diff
@@ -6,12 +6,20 @@ body {
     font-family: 'DM Sans',sans-serif;
 
 }
-
+.search-input::placeholder::before {
+    content: "\f002"; /* Unicode value for the search icon in Font Awesome */
+    font-family: "Font Awesome 5 Free";
```

Comment:
what is Font Awesome 5 free font family?? Please understand the code before copy pasting